k8s/conf/spire/server-configmap.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spire-server
+  namespace: spire
+data:
+  server.conf: |
+    server {
+      bind_address = "0.0.0.0"
+      bind_port = "8081"
+      trust_domain = "test.com"
+      data_dir = "/run/spire/data"
+      log_level = "DEBUG"
+      svid_ttl = "1h"
+      upstream_bundle = true
+      ca_subject = {
+        Country = ["US"],
+        Organization = ["SPIFFE"],
+        CommonName = "",
+      }
+    }
+    plugins {
+      DataStore "sql" {
+        plugin_data {
+          database_type = "sqlite3"
+          connection_string = "/run/spire/data/datastore.sqlite3"
+        }
+      }
+      NodeAttestor "k8s_sat" {
+        plugin_data {
+          clusters = {
+            # NOTE: Change this to your cluster name
+            "kubernetes" = {
+              use_token_review_api_validation = true
+              service_account_whitelist = ["spire:spire-agent"]
+            }
+          }
+        }
+      }
+      NodeResolver "noop" {
+        plugin_data {}
+      }
+      KeyManager "disk" {
+        plugin_data {
+          keys_path = "/run/spire/data/keys.json"
+        }
+      }
+      UpstreamCA "disk" {
+        plugin_data {
+          ttl = "12h"
+          key_file_path = "/run/spire/secrets/bootstrap.key"
+          cert_file_path = "/run/spire/config/bootstrap.crt"
+        }
+      }
+    }
+  bootstrap.crt: |
+    -----BEGIN CERTIFICATE-----
```

Comment:
as discussed this certificates will be removed and generated using helm in next PRs